src/main/resources/assets/patchouli/lang/de_DE.lang

```diff
@@ -0,0 +1,93 @@
+# Item Names
+item.patchouli:guide_book.name=Reiseführer
```

Comment:
I don't think this properly represents the meaning of the name.

Reply:
Hmm... "Anleitungsbuch" would be better I think.